### ISSUE TITLE
Why does it keep trying to install a new version of 'yaml'?? 

### ISSUE THREAD

**Issue by genobobeno** [ramnathv/rCharts#736]
yaml is already installed on my computer. There's no reason rCharts should need to uninstall and reinstall a package I already have.

**Comment by genobobeno** [comment]
Note, a reinstallation of R fixes the yaml problem. rCharts literally breaks the base package yaml. I think it's obvious that this package is no longer being supported.

**Comment by region-spotteR** [comment]
yep. Trying to install this package had the effect you described. Now I need to reinstall all my packages again. At least this way I updated my R version. 

In addition I was a bit surprised how little documentation there was. I had hoped for some styling options. But the package is very basic.
